source/codegen/metadata/nidmm/functions.py

```diff
@@ -0,0 +1,2171 @@
+        'returns': 'ViStatus'
+    },
+    "GetMeasurementPeriod": {
+        "parameters": [
+        {
+            "name": "vi",
+            "direction": "in",
+            "type": "ViSession"
+        },
+        {
+            "name": "period",
+            "direction": "out",
+            "type": "ViReal64"
+        }
+        ],
+        "returns": "ViStatus"
+    },
+    # "GetNextCoercionRecord": {
+    #     "parameters": [
+    #     {
+    #         "name": "vi",
+    #         "direction": "in",
+    #         "type": "ViSession"
+    #     },
+    #     {
+    #         "name": "bufferSize",
+    #         "direction": "in",
+    #         "type": "ViInt32"
+    #     },
+    #     {
+    #         "name": "coercionRecord",
+    #         "direction": "out",
+    #         "type": "ViChar[]"
+    #     }
+    #     ],
+    #     "returns": "ViStatus"
+    # },
+    # "GetNextInterchangeWarning": {
+    #     "parameters": [
+    #     {
+    #         "name": "vi",
+    #         "direction": "in",
+    #         "type": "ViSession"
+    #     },
+    #     {
+    #         "name": "bufferSize",
+    #         "direction": "in",
+    #         "type": "ViInt32"
+    #     },
+    #     {
+    #         "name": "interchangeWarning",
+    #         "direction": "out",
+    #         "type": "ViChar[]"
+    #     }
+    #     ],
+    #     "returns": "ViStatus"
+    # },
+    'GetSelfCalSupported': {
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'vi',
+                'type': 'ViSession'
+            },
+            {
+                'direction': 'out',
+                'name': 'selfCalSupported',
+                'type': 'ViBoolean'
+            }
+        ],
+        'returns': 'ViStatus'
+    },
+    'ImportAttributeConfigurationBuffer': {
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'vi',
+                'type': 'ViSession'
+            },
+            {
+                'direction': 'in',
+                'name': 'size',
+                'type': 'ViInt32'
+            },
+            {
+                'direction': 'in',
+                'name': 'configuration',
+                'python_api_converter_name': 'convert_to_bytes',
+                'size': {
+                    'mechanism': 'len',
+                    'value': 'size'
+                },
+                'type': 'ViInt8[]',
+                'type_in_documentation': 'bytes'
+            }
+        ],
+        'returns': 'ViStatus'
+    },
+    'ImportAttributeConfigurationFile': {
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'vi',
+                'type': 'ViSession'
+            },
+            {
+                'direction': 'in',
+                'name': 'filePath',
+                'type': 'ViConstString'
+            }
+        ],
+        'returns': 'ViStatus'
+    },
+    # "InitExtCal": {
```

Comment:
Same as the other comment about checking in commented code. Should just remove this function from functions.py if we don't want to include it.

Reply:
removed